.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
```

Comment:
Do we have a common shared eslint config somewhere?

Reply:
@hatched we don't have, at least as far as I know. Perhaps we should create one. Will add this topic to the agenda for the next Dev catch-up.